src,lib: introduce `util.getSystemErrorMessage(err)` (#54075)
nodejs/node: doc/api/util.md

});
```

## `util.getSystemErrorMessage(err)`

<!-- YAML
added: REPLACEME
-->

* `err` {number}
* Returns: {string}

Returns the string message for a numeric error code that comes from a Node.js
API.

> The mapping between error codes and error names is platform-dependent.
> See [Common System Errors][] for the names of common errors.

Reviewer: ```suggestion
The mapping between error codes and string messages is platform-dependent.
```

Author: Thanks, fixed!
